synctex warns on each and every file that doesn't have a respective synctex file,
remove the warning
code from https://gitlab.gnome.org/GNOME/evince/commit/678410e81d0c889f4db4e995ca451ed62b8a2eee
Debian bug #908964
---
 texk/web2c/synctexdir/synctex_parser.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- texlive-bin.orig/texk/web2c/synctexdir/synctex_parser.c
+++ texlive-bin/texk/web2c/synctexdir/synctex_parser.c
@@ -6023,7 +6023,8 @@ synctex_scanner_p synctex_scanner_new_wi
     if ((scanner->reader = synctex_reader_init_with_output_file(scanner->reader, output, build_directory))) {
         return parse? synctex_scanner_parse(scanner):scanner;
     }
-    _synctex_error("No file?");
+    // don't warn if no file is present
+    // _synctex_error("No file?");
     return NULL;
 }
 
